File: src/main/java/no/nav/service/pensjon/regler/to/VilkarsprovHalvpensjonRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.Date;

import no.nav.domain.pensjon.regler.kode.GrunnlagsrolleCti;
import no.nav.domain.pensjon.regler.krav.Kravhode;
import no.nav.domain.pensjon.regler.vedtak.VilkarsVedtak;


/**
 * @author Magnus Bakken (Accenture) PK-10597 Beslutningsstøtte for vilkår
 */
public class VilkarsprovHalvpensjonRequest extends ServiceRequest {
	
	private Kravhode kravhode;
	private Date fom;
	private Date tom;
	private GrunnlagsrolleCti grunnlagsrolle;

    /**
     * Vilkårsvedtak for uføretrygd.
     */
    private VilkarsVedtak vilkarsvedtak;

}

File: src/main/kotlin/no/nav/pensjon/regler/to/VilkarsprovHalvpensjonRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.enum.GrunnlagsrolleEnum
import no.nav.pensjon.regler.domain.krav.Kravhode
import no.nav.pensjon.regler.domain.vedtak.VilkarsVedtak
import java.util.*
import java.time.LocalDate

class VilkarsprovHalvpensjonRequest : ServiceRequest() {
    var kravhode: Kravhode? = null
    var fomLd: LocalDate? = null
    var tomLd: LocalDate? = null
    var grunnlagsrolleEnum: GrunnlagsrolleEnum? = null

    /**
     * vilkårsvedtak for uføretrygd.
     */
    var vilkarsvedtak: VilkarsVedtak? = null
}